doc/manual/source/developer_guide/ModificationIntro.rst

```diff
@@ -30,21 +30,23 @@ at least read in order, as doing complex things with the code require the
 ability to do the simpler things.
 
 We are very happy to accept patches, features, and bugfixes from any member of
-the community!  Enzo is developed using mercurial, primarily because it enables
+the community!  Enzo is developed using GitHub, primarily because it enables
 very easy and straightforward submission of changesets.  We're eager to hear
 from you, and if you are developing Enzo, please subscribe to the users'
 mailing list:
 
 http://groups.google.com/group/enzo-users
 
-This document describes how to use Mercurial to make changes to Enzo, how to
+This document describes how to use GitHub to make changes to Enzo, how to
 send those changes upstream, and how to navigate the Enzo source tree.
 
 .. highlight:: none
 
 Mercurial Introduction
 ----------------------
 
+[This should be updated to refer to Git.]
```

Comment:
why not just update these?

Reply:
I just started using git so I think someone more experienced should write an introduction for how to use it. I'll add a note to the user that this document will be updated in the future since Enzo is no longer hosted on BitBucket and does not use Mercurial.

Reply:
Seems reasonable to me!